engine/execution/computation/computer/computer.go

```diff
@@ -226,6 +228,18 @@ func (e *blockComputer) executeBlock(
 	res.Proofs = proofs
 	res.TrieUpdates = trieUpdates
 
+	elapsedTimeNS := time.Since(startedAt)
+	e.log.Info().Hex("block_id", logging.Entity(blockCtx.BlockHeader)).
+		Int("numberOfTransactions", len(res.TransactionResults)).
+		Int64("timeSpentInNS", elapsedTimeNS.Nanoseconds()).
+		Timestamp().
+		Msg("Block executed")
+
+	msg := "***  Block executed in: " + elapsedTimeNS.String() + "  ***"
```

Comment:
Put this printout in the `computer.go` file for now, but we can move it to the test file if that feels like better organization.